shared/js/remoteCursorStore.js

```diff
@@ -66,14 +75,36 @@ loop.store.RemoteCursorStore = (function() {
     },
 
     /**
-     * Receives cursor data.
+     * Sends the cursor data to the SDK for broadcasting.
+     *
+     * @param {sharedActions.SendCursorData}
+     *  actionData Contains the updated information for the cursor's position
+     *      {
+     *       ratioX {[0-1]} Cursor's position on the X axis
+     *       ratioY {[0-1]} Cursor's position on the Y axis
+     *       type   {String} Type of the data being sent
+     *      }
+     */
+    sendCursorData: function(actionData) {
+      switch (actionData.type) {
+        case CURSOR_MESSAGE_TYPES.POSITION:
+          this._sdkDriver.sendCursorMessage(actionData);
+          break;
+      }
+    },
+
+    /**
+     * Receives cursor data and updates the store.
      *
      * @param {sharedActions.receivedCursorData} actionData
      */
     receivedCursorData: function(actionData) {
       switch (actionData.type) {
         case CURSOR_MESSAGE_TYPES.POSITION:
-          // TODO: handle cursor position if it's desktop instead of standalone
+          if (this._isDesktop) {
+            loop.request("AddRemoteCursorOverlay", actionData);
+          }
+
```

Comment:
Actually, it probably makes more sense to have `conversation.jsx` `AppControllerView` `listenTo` `change:remoteCursorPosition` as it also has access to `cursorStore`.